sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -135,6 +142,30 @@ export class CancelApplicationAssessmentProcessor {
         assessment.id,
         transactionEntityManager,
       );
+      await summary.info("Overawards rollback check executed.");
+      if (
```

Comment:
A comment here will be helpful to understand the context that the logic must be executed during `Cancel application` but not resubmit application.

Reply:
Comment added.